README.md

```diff
@@ -1 +1,64 @@
-# microverse-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio 
+
+Project: Build my personal portfolio.
+The project is inspired by the figma microverse template 1 (https://figma.com/)
+It use to learn how parse a Figma design to create a UI, and use Flexbox to place elements in the page.
+
+
+**Mobile Version**
+
+![screenshot](./portfolio-mobile.PNG)
+
+ **Web version**
+
+![screenshot](./portfolio-web.PNG)
+ 
+
+## Demo 
+
+[Live Demo Link](https://portfoliocodecaiine.github.io/)
+ 
+## Stacks
+
+- git/github
+- html/css
+- linters (html - css )
+
+## Get Started
+
+- Clone this project on your local environment 
+- Get your browser update 
+
+
```

Comment:
- [ ] While this is a good start, I think it would be better if you gave the user more detailed instructions on how to run your project. For example instead of saying ` Clone this project on your local environment `, you could say `run git clone <URL> in your terminal` this is clearer. Does the user need to cd into a folder? which file will they run to open the server? Kindly work on this